src/auto-complete.jsx

```diff
@@ -26,7 +27,7 @@ const AutoComplete = React.createClass({
     /**
```

Comment:
Having a look at the `Popover` component, the `animated` property doesn't seem useful. @chrismcv Any idea?